test/e2e/helpers.go

```diff
@@ -71,3 +76,80 @@ func WaitForSuccessfulDeployment(t *testing.T, f *test.Framework, deployment app
 	t.Logf("Deployment %s has been initialized successfully\n", deployment.Name)
 	return nil
 }
+
+// createRuntimeObject creates a runtime object using the test framework
+func createRuntimeObject(f *test.Framework, ctx *test.TestCtx, obj runtime.Object) error {
+	return f.Client.Create(
+		context.TODO(),
+		obj,
+		&test.CleanupOptions{
+			TestContext:   ctx,
+			Timeout:       cleanupTimeout,
+			RetryInterval: cleanupRetryInterval,
+		})
+}
+
+// deleteRuntimeObject deletes a runtime object and wait for up to 1 minute for the
+// object to be removed from the cluster
+func deleteRuntimeObject(f *test.Framework, obj runtime.Object, name string, namespace string) error {
+	// Delete the runtime object
+	err := f.Client.Delete(
+		context.TODO(),
+		obj)
+	if err != nil {
+		return fmt.Errorf("Error deleting obj: %v", err)
+	}
+
+	// Confirm that the runtime object was deleted
+	namespacedName := types.NamespacedName{Name: name, Namespace: namespace}
```

Comment:
Why is this function also confirming it was deleted while `createRuntimeObject` does only the creation? What if I have a use where I just want to delete an object and not care if it actually got deleted?

Reply:
After reviewing a comment you made earlier[1] this function was removed given that the test is not required to delete a csc.